### DISCUSSION TITLE
output files not removed when running part2_01_pstfrom_pest_setup notebook

### DISCUSSION THREAD

**Discussion by kes79** [gmdsi/GMDSI_notebooks#119] [Q&A]
Hi, I am getting an error when running this tutorial (and in my code based off this tutorial - eek) that the output files are not removed with forward_run.py, and resulting in the "error removing tmp file:heads.csv" etc occuring. Any ideas what I might be doing wrong??

**Comment by jtwhite79** [comment]
Can you post a screenshot/terminal output for the error?  The forward run script should try to remove those temp files and it will report if it encounters an error (with a message similar to what your indicate) but will continue..maybe this is what you are seeing?

**Reply by kes79** [reply]
Thanks, this is what I am seeing, it will continue, but I am not sure if the files are being deleted correctly or not: 

![output](https://github.com/gmdsi/GMDSI_notebooks/assets/60296993/0f099115-d8f4-4ba6-9de2-0be572060492)
